doc/manual/expressions/builtins.xml

```diff
@@ -280,8 +280,15 @@ with import (fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/nixo
 
 stdenv.mkDerivation { … }
 </programlisting>
-
-    Note that when obtaining the hash with <varname>nix-prefetch-url
+    <para>The fetched tarball is chached for a certain amount of time 
+    (1 hour by default) in <literal>~/.cache/nix/tarballs/</literal>.
```

Comment:
(a nit, you don't have to change this:) `<literal>` could be [`<filename>`](https://tdg.docbook.org/tdg/5.2/filename.html).